packages/oats/src/generate-types.ts

```diff
@@ -696,16 +708,16 @@ export function run(options: Options) {
             ts.NodeFlags.Const
           )
         ),
-        ts.factory.createReturnStatement(
-          ts.createConditional(
```

Comment:
`ts.createConditional` created a ternary expression
it doesn't seem to have been replaced with anything (despite the docs saying so, but they point to a non-existing method) so I used a good ol' if statement instead